chronos/timer.nim

```diff
@@ -8,45 +8,141 @@
 #    Apache License, version 2.0, (LICENSE-APACHEv2)
 #                MIT license (LICENSE-MIT)
 
-
 ## This module implements cross-platform system timer with
 ## milliseconds resolution.
+##
+## Timer supports two types of clocks:
+## ``fast`` uses the most fast OS primitive to obtain wall clock time.
```

Comment:
let's call this system time perhaps, and highlight the risk of it going back in time..